### ISSUE TITLE
Baritone not knowing were to go and wandering around randomly

### ISSUE THREAD

**Issue by Absurv69** [cabaletta/baritone#3874]
https://ibb.co/H77mB4v

There is an image of the issue, i am loading a schematic and baritone keeps wandering around (to the green blocks u can see it wants to go to) and once it reaches them just turns around as there is nothing wrong with those blocks and then gets stuck in an infinite loop of walking around.

Note i have temporarily excluded the carpets in the map art i am making so i can build the foundations first, but it is clear that baritone isn't working as intended as you can see the translucent blocks to the left in the image (Unfinished parts of the schematic that need to be built) and despite being next to it, decides that it isn't worth its time and goes of to wander around.

**Comment by Absurv69** [comment]
No build in layers is off as i am making map art.

**Comment by rycbar0** [comment]
still waiting for your `#modified` settings. also what command are you running.

**Comment by Absurv69** [comment]
running litematica 
Modifed settings are acceptablethrowawayitems, allow inventory, blockreachdistance, blockstoavoid, blockstoavoidbreaking, blockstodisallowbreaking, buildignoreblocks, buildskipblocks, mapartmode. (Note i have tried with mapartmode off and on)

**Comment by wagyourtail** [comment]
How big is this schematic?

**Comment by h3tz** [comment]
#explore ?

**Comment by ZacSharp** [comment]
> #explore ?

No, that's an unrelated command and won't help here.

I do wonder though whether those green boxes were out of render distance when `#build` was executed (and whether they are in the -x -z corner of the build).

**Comment by h3tz** [comment]
> > #explore ?
> 
> No, that's an unrelated command and won't help here.
> 
> I do wonder though whether those green boxes were out of render distance when `#build` was executed (and whether they are in the -x -z corner of the build).

not really. The explore implementation already solved your issue. So double check that code again.

**Comment by wagyourtail** [comment]
Read their issue again. Their issue is that its walkinng around randomly when it shouldnt be

**Comment by k6492vzw58-cyber** [comment]
Sorry to wake this up 3 years later. I’m having the SAME exact issue while creating map arts. Any fix to this? And yes it’s consistent with pathing to blocks within the schematic but also inside of unloaded chunks. If anybody sees this id love a helping hand. Thank you :)
